### ISSUE TITLE
a GravityPagerSnapHelper snap bug in version 1.5

### ISSUE THREAD

**Issue by huangjim** [rubensousa/RecyclerViewSnap#56]
I am still using version 1.5 because my project does not surpport androidx. I need to use GravityPagerSnapHelper and listening page selected,like this:
`
 new GravityPagerSnapHelper(Gravity.START, true, new GravitySnapHelper.SnapListener() {
                @Override
                public void onSnap(int position) {
                    Toast.makeText(holder.recyclerView.getContext(),"Snapped: "+ position,Toast.LENGTH_SHORT).show();
                }
            }).attachToRecyclerView(holder.recyclerView);
`
it works well before last one. When I slide to the last one, it always automatically returns to the previous one. Is there any way to solve this problem?

**Comment by rubensousa** [comment]
Set "enableSnapLastItem" to false in the constructor instead.

**Comment by huangjim** [comment]
i make a gif
![ ](https://im.ezgif.com/tmp/ezgif-1-9ae145611902.gif)
if i set "enableSnapLastItem" to false in the constructor, it is fail to listen the callback of the last one

**Comment by rubensousa** [comment]
That URL is returning 404. I think that issue was fixed on 2.0. I'll check if it happens on 2.0 and if it doesn't I'll backport the fix to the old support libraries.

**Comment by huangjim** [comment]
You are right. It does not happen on 2.0. I am looking forward to fix this bug. Thank you very much.

**Comment by huangjim** [comment]
> That URL is giving 404. I think that issue was fixed on 2.0. I'll check if it happens on 2.0 and if it doesn't I'll backport the fix to the old support libraries.

![ ](http://zhimin-image.oss-cn-shenzhen.aliyuncs.com/18-11-11/ezgif.com-video-to-gif.gif)

**Comment by rubensousa** [comment]
It's still not in jcenter, but here it is:

```
implementation 'com.github.rubensousa:gravitysnaphelper-compat:2.0'
```
